docs/notebooks/formulary/solar_plasma_beta.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<p>I would use "strongly" rather than "hugely" in third sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/1552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>